contrib/docs/tutorials/help-im-behind-a-corporate-proxy.md

```diff
@@ -24,38 +24,105 @@ There are however some ways to get this to work without too much effort.
 
 1. Go to the entry file for the backend (typically `packages/backend/src/index.ts`), and add the following at the top:
 
-   ```ts
-   import 'global-agent/bootstrap';
-   import { setGlobalDispatcher, ProxyAgent } from 'undici';
-
-   const proxyEnv =
-     process.env.GLOBAL_AGENT_HTTP_PROXY ||
-     process.env.GLOBAL_AGENT_HTTPS_PROXY;
-
-   if (proxyEnv) {
-     const proxyUrl = new URL(proxyEnv);
-     setGlobalDispatcher(
-       new ProxyAgent({
-         uri: proxyUrl.protocol + proxyUrl.host,
-         token:
-           proxyUrl.username && proxyUrl.password
-             ? `Basic ${Buffer.from(
-                 `${proxyUrl.username}:${proxyUrl.password}`,
-               ).toString('base64')}`
-             : undefined,
-       }),
-     );
-   }
-   ```
-
-   The first import automatically bootstraps `global-agent`, which addresses `node-fetch` proxying. The lines of code below that peeks into the same environment variables as `global-agent` uses, and also leverages them to set up the `undici` package which affects native `fetch`. Does that seem weird? Yes, we think so too. But in the current state of the Node.js ecosystem, that's how it works.
-
-   This code is kept brief for illustrative purposes. You may want to adjust it slightly if you need support for [no-proxy excludes](https://gist.github.com/zicklag/1bb50db6c5138de347c224fda14286da) or only do proxying in local development etc. Also see [the `global-agent` docs](https://github.com/gajus/global-agent) for information about its configuration options.
+````ts
```

Comment:
I think you accidentally made four backticks instead of three here and at the end of the code as well

Reply:
It is not accidental. 4-backticks pattern allows for embedding triple-backticks into the code. See https://stackoverflow.com/a/71150969.

There is a TSDoc with a triple backtick inside the code block.